beetsplug/acoustic.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import (division, absolute_import, print_function,
```

Comment:
Added in https://github.com/sampsyo/beets/commit/29e6cbedfe4fa3eb31ca033ef776c98eafa605c6